prog/storage/setup_spdk.rb

```diff
@@ -36,7 +36,7 @@ def self.assemble(vm_host_id, version, start_service: false, allocation_weight:
       allocation_weight: 0,
       vm_host_id: vm_host.id,
       cpu_count: vm_host.spdk_cpu_count,
-      hugepages: 3
```

Comment:
Yesterday, I noticed that we still check if the host has 2 hugepages, even though we allocated 3. We probably forgot to update it when we increased it to 3. We need to make that update. We could extract it to a variable and use that variable for both checks.

```ruby
    fail "No available hugepages" if frame["start_service"] && vm_host.used_hugepages_1g > vm_host.total_hugepages_1g - 2
```